### ISSUE TITLE
html.exploit.cve_2016_7201-1

### ISSUE THREAD

**Issue by jcscott2015** [compat-table/compat-table#1220]
I haven't seen this reported in a search of issues, but your compat-table site seems to be infected or at least vulnerable to the html.exploit.cve_2016_7201-1. This exploit affects Windows users. I'm on a Mac using ClamXAV, and the exploit always pops up when visiting https://kangax.github.io/compat-table/es6/. This is likely a server issue.

**Comment by chicoxyzzy** [comment]
It's because of Flattr and / or other 3rd-party scripts.

@kangax could we remove some of them?

**Comment by ljharb** [comment]
Can you link to the exploit? What's the issue?

**Comment by kangax** [comment]
Yeah, I don't mind removing them if that's the issue. But shouldn't it be reported to Flattr then? I'm also unclear on how exactly they're affected?

**Comment by chicoxyzzy** [comment]
Exploit description https://nvd.nist.gov/vuln/detail/CVE-2016-7201

Actually it seems that the problem with memory corruption could be on our side. @jcscott2015 please more info if that's possible.

**Comment by ljharb** [comment]
The exploit tho is in *Edge*, not in our JS; Edge users have an action item here, but why do we?

**Comment by chicoxyzzy** [comment]
IMO we can fix problem on our side too if it's easy enough

**Comment by ljharb** [comment]
There's nothing to fix tho ; the vulnerability is Edge users who visit a malicious website. We aren't one, and neither is Flattr, and those users should update their Edge anyways. Am I reading this CVE wrong?

**Comment by jcscott2015** [comment]
@chicoxyzzy I don't have much to add. I usually use Chrome Version 62.0.3202.94 (Official Build) (64-bit) on Mac, but Safari triggers the same infection alert. On page load, Chrome dumps a small file (247 kb) into the ClamXAV quarantine folder, while Safari dumps the whole web page (4.3 mgs).

I'm not having any memory issues with cache -- nothing in the logs. There's definitely a payload. I've attached a ZIP of one of the Chrome files.

[f_00295b.zip](https://github.com/kangax/compat-table/files/1469039/f_00295b.zip)
